databases/source/mysql/attributes.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Attributes
+description: Supported MySQL Column Attributes in SchemaHero
+---
+
+The goal of SchemaHero is to support all additional Mysql column attributes. 
+If there are any missing, [open an issue](https://github.com/schemahero/schemahero/issues/new) to report it.
+
+| Column Attribute | SchemaHero Version |
+|-----------------|--------------------|
+| AUTO_INCREMENT | v.0.X.X |
```

Comment:
?

Reply:
sure.  wiill you add to the reference doc also:  https://schemahero.io/reference/v1alpha4/table/

Reply:
i did https://github.com/schemahero/schemahero-docs/pull/31/files#diff-b82fa5c5edc5f2ab311c2dc59f294d97R54